docs/developers/README.md

```diff
@@ -0,0 +1,54 @@
+## Developer docs
```

Comment:
my very last request is that we have a link to these docs somewhere - maybe from the DEVELOPMENT.md and/or from the main README :pray: 

i promise to stop the scope creep after this :heart: